app/controllers/v0/education_benefits_claims_controller.rb

```diff
@@ -0,0 +1,23 @@
+module V0
```

Comment:
I went down the route of doing v1/rx initially, but I tend to agree that rx/v1 makes a bit more sense... for example if MHV changes stuff up completely on their backend in some new version to support new features, we don't want to bump versions for all other apis, just the ones for prescriptions.